app/models/gsa18f/procurement.rb

```diff
@@ -10,6 +10,13 @@ class Procurement < ActiveRecord::Base
     URGENCY = DATA['URGENCY']
     OFFICES = DATA['OFFICES']
     RECURRENCE = DATA['RECURRENCE']
+    PURCHASE_TYPES = {
+      "Software" => 0,
+      "Training/Event" => 1,
+      "Office Supply/Miscellaneous" => 2
+    }
```

Comment:
Changing them without thinking about it would be an easy mistake to make, and would be a nasty data inconsistency problem after the fact, so I'd err on the side of caution.